tests/client/main.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define CATCH_CONFIG_MAIN
+
+#include "api_internal.h"
+#include "catch2\catch.hpp"
+#include "ebpf_api.h"
+#pragma warning(push)
+#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
+                                // 'type2', possible loss of data
+#undef VOID
+#include "ebpf_verifier.hpp"
+#pragma warning(pop)
+#include "header.h"
+#include "rpc_client.h"
+#include "rpc_interface_h.h"
+#include "service_helper.h"
+
+#define EBPF_SERVICE_BINARY_NAME L"ebpfsvc.exe"
+#define EBPF_SERVICE_NAME L"ebpfsvc"
+
+service_install_helper service_helper(EBPF_SERVICE_NAME, EBPF_SERVICE_BINARY_NAME);
```

Comment:
can this be static? (development guide says "DO use const and static and visibility modifiers to scope exposure of variables and methods as much as possible.")

Reply:
changed to static.